diff -up xmms2-0.5DrLecter/wscript.versionsanity xmms2-0.5DrLecter/wscript
--- xmms2-0.5DrLecter/wscript.versionsanity	2008-12-10 15:22:56.000000000 -0500
+++ xmms2-0.5DrLecter/wscript	2008-12-10 15:27:09.000000000 -0500
@@ -23,7 +23,7 @@ import Object
 import Utils
 import Common
 
-BASEVERSION="0.5 DrLecter"
+BASEVERSION="0.5"
 APPNAME='xmms2'
 
 srcdir='.'
@@ -252,14 +252,15 @@ def configure(conf):
 
     nam,changed = gittools.get_info()
     conf.check_message("git commit id", "", True, nam)
-    if Params.g_options.customversion:
-        conf.env["VERSION"] = BASEVERSION + " (%s + %s)" % (nam, Params.g_options.customversion)
-    else:
-        dirty=""
-        if changed:
-            dirty="-dirty"
-        conf.check_message("uncommitted changes", "", bool(changed))
-        conf.env["VERSION"] = BASEVERSION + " (git commit: %s%s)" % (nam, dirty)
+    ## if Params.g_options.customversion:
+    ##     conf.env["VERSION"] = BASEVERSION + " (%s + %s)" % (nam, Params.g_options.customversion)
+    ## else:
+    ##     dirty=""
+    ##    if changed:
+    ##        dirty="-dirty"
+    ##    conf.check_message("uncommitted changes", "", bool(changed))
+    ##    conf.env["VERSION"] = BASEVERSION + " (git commit: %s%s)" % (nam, dirty)
+    conf.env["VERSION"] = BASEVERSION
 
     ## conf.env["CCFLAGS"] = Utils.to_list(conf.env["CCFLAGS"]) + ['-g', '-O0']
     ## conf.env["CXXFLAGS"] = Utils.to_list(conf.env["CXXFLAGS"]) + ['-g', '-O0']
