diff -up xmms2-0.6DrMattDestruction/src/include/xmms/wscript.BAD xmms2-0.6DrMattDestruction/src/include/xmms/wscript
--- xmms2-0.6DrMattDestruction/src/include/xmms/wscript.BAD	2009-08-11 15:17:57.020242949 -0400
+++ xmms2-0.6DrMattDestruction/src/include/xmms/wscript	2009-08-11 15:18:04.940381687 -0400
@@ -15,8 +15,7 @@ def configure(conf):
 
     defs = {}
 
-    defs['PKGLIBDIR'] = os.path.join(conf.env['PREFIX'],
-                                     'lib', 'xmms2')
+    defs['PKGLIBDIR'] = os.path.join(conf.env['LIBDIR'], 'xmms2')
     defs['BINDIR']    = conf.env['BINDIR']
     defs['SHAREDDIR'] = os.path.join(conf.env['PREFIX'],
                                      'share', 'xmms2')
diff -up xmms2-0.6DrMattDestruction/waftools/tool.py.BAD xmms2-0.6DrMattDestruction/waftools/tool.py
--- xmms2-0.6DrMattDestruction/waftools/tool.py.BAD	2009-08-11 15:28:30.157242468 -0400
+++ xmms2-0.6DrMattDestruction/waftools/tool.py	2009-08-11 15:28:45.251245035 -0400
@@ -6,5 +6,5 @@ def add_install_flag(bld, obj):
 
     if env['explicit_install_name']:
         libname = obj.env["shlib_PATTERN"] % obj.target
-        insname = os.path.join(obj.env["PREFIX"], 'lib', libname)
+        insname = os.path.join(obj.env["LIBDIR"], libname)
         obj.env.append_unique("LINKFLAGS", '-install_name ' + insname)
diff -up xmms2-0.6DrMattDestruction/wafadmin/Tools/libtool.py.BAD xmms2-0.6DrMattDestruction/wafadmin/Tools/libtool.py
--- xmms2-0.6DrMattDestruction/wafadmin/Tools/libtool.py.BAD	2009-08-11 15:33:10.276383650 -0400
+++ xmms2-0.6DrMattDestruction/wafadmin/Tools/libtool.py	2009-08-11 15:33:33.646236480 -0400
@@ -45,7 +45,7 @@ def fakelibtool_build(task):
 	fu("current=0\n")
 	fu("age=0\nrevision=0\ninstalled=yes\nshouldnotlink=no\n")
 	fu("dlopen=''\ndlpreopen=''\n")
-	fu("libdir='%s/lib'\n" % env['PREFIX'])
+	fu("libdir='%s'\n" % env['LIBDIR'])
 	dest.close()
 	return 0
 
@@ -75,7 +75,7 @@ def apply_link_libtool(self):
 		self.latask = latask
 
 	if Options.commands['install'] or Options.commands['uninstall']:
-		Build.bld.install_files('PREFIX', 'lib', linktask.outputs[0].abspath(self.env), self.env)
+		Build.bld.install_files('LIBDIR', linktask.outputs[0].abspath(self.env), self.env)
 
 @taskgen
 @feature("libtool")
diff -up xmms2-0.6DrMattDestruction/wafadmin/Tools/ccroot.py.BAD xmms2-0.6DrMattDestruction/wafadmin/Tools/ccroot.py
--- xmms2-0.6DrMattDestruction/wafadmin/Tools/ccroot.py.BAD	2009-08-11 15:35:28.864257981 -0400
+++ xmms2-0.6DrMattDestruction/wafadmin/Tools/ccroot.py	2009-08-11 15:35:36.165270276 -0400
@@ -127,7 +127,7 @@ def vars_target_cprogram(self):
 @feature('cstaticlib', 'dstaticlib', 'cshlib', 'dshlib')
 @before('apply_core')
 def vars_target_cstaticlib(self):
-	self.default_install_path = '${PREFIX}/lib'
+	self.default_install_path = '${LIBDIR}'
 	if sys.platform in ['win32', 'cygwin']:
 		# on win32, libraries need the execute bit, else we
 		# get 'permission denied' when using them (issue 283)
diff -up xmms2-0.6DrMattDestruction/wafadmin/Tools/gnu_dirs.py.BAD xmms2-0.6DrMattDestruction/wafadmin/Tools/gnu_dirs.py
--- xmms2-0.6DrMattDestruction/wafadmin/Tools/gnu_dirs.py.BAD	2009-08-11 15:35:49.334273249 -0400
+++ xmms2-0.6DrMattDestruction/wafadmin/Tools/gnu_dirs.py	2009-08-11 15:36:35.058242288 -0400
@@ -39,7 +39,7 @@ libexecdir, program executables, $(EXEC_
 sysconfdir, read-only single-machine data, $(PREFIX)/etc
 sharedstatedir, modifiable architecture-independent data, $(PREFIX)/com
 localstatedir, modifiable single-machine data, $(PREFIX)/var
-libdir, object code libraries, $(EXEC_PREFIX)/lib
+libdir, object code libraries, $(LIBDIR)
 includedir, C header files, $(PREFIX)/include
 oldincludedir, C header files for non-gcc, /usr/include
 datarootdir, read-only arch.-independent data root, $(PREFIX)/share
