diff -up xmms2-0.5DrLecter/src/include/xmms/wscript.BAD xmms2-0.5DrLecter/src/include/xmms/wscript
--- xmms2-0.5DrLecter/src/include/xmms/wscript.BAD	2008-12-04 16:28:12.000000000 -0500
+++ xmms2-0.5DrLecter/src/include/xmms/wscript	2008-12-04 16:28:44.000000000 -0500
@@ -12,8 +12,7 @@ def configure(conf):
 
     defs = {}
 
-    defs['PKGLIBDIR'] = os.path.join(conf.env['PREFIX'],
-                                     'lib', 'xmms2')
+    defs['PKGLIBDIR'] = os.path.join(conf.env['LIBDIR'], 'xmms2')
     defs['BINDIR']    = conf.env['BINDIR']
     defs['SHAREDDIR'] = os.path.join(conf.env['PREFIX'],
                                      'share', 'xmms2')
diff -up xmms2-0.5DrLecter/wafadmin/Tools/python.py.BAD xmms2-0.5DrLecter/wafadmin/Tools/python.py
--- xmms2-0.5DrLecter/wafadmin/Tools/python.py.BAD	2008-12-04 16:36:29.000000000 -0500
+++ xmms2-0.5DrLecter/wafadmin/Tools/python.py	2008-12-04 16:36:41.000000000 -0500
@@ -211,7 +211,7 @@ def check_python_version(conf, minver=No
 			else:
 				python_LIBDEST = None
 			if python_LIBDEST is None:
-				python_LIBDEST = os.path.join(conf.env['PREFIX'], "lib", "python" + pyver)
+				python_LIBDEST = os.path.join(conf.env['LIBDIR'], "python" + pyver)
 			dir = os.path.join(python_LIBDEST, "site-packages")
 
 		conf.add_define('PYTHONDIR', dir)
